keps/sig-cluster-lifecycle/kubeadm/20190227-artifact-generation.md

```diff
@@ -16,7 +16,7 @@ approvers:
   - "@timothysc"
 editor: "@Klaven"
 creation-date: 2019-02-15
-last-updated: yyyy-mm-dd
+last-updated: 2019-06-27
```

Comment:
should be feb 27 2019 with
24adcec58daaf280c7377becb7277adb6683c726

@Klaven do you have a typo in your git config? I see your last name may be missing an `o` perhaps?

`git log -1 --format=%an 24adcec58daaf280c7377becb7277adb6683c726`